### PR TITLE
added hebrew translations

### PR DESCRIPTION
**Pull request by kevinpapst**
## Description

Thanks to @eransurg

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist
- [x] I updated the documentation (see [here](https://github.com/kevinpapst/AdminLTEBundle/tree/master/Resources/docs))
- [x] I agree that this code is used in AdminLTEBundle and will be published under the [MIT license](https://github.com/kevinpapst/AdminLTEBundle/blob/master/LICENSE)

### CODE REVIEW

File: Resources/translations/AdminLTEBundle.he.xliff [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+                <source>Remember Me</source>
+                <target>זכור אותי</target>
+            </trans-unit>
+            <trans-unit id="Sign in to start your session">
+                <source>Sign in to start your session</source>
+                <target>התחבר</target>
+            </trans-unit>
+            <trans-unit id="Sign In">
+                <source>Sign In</source>
+                <target>חבר</target>
```
Comment (eransurg) [inline Resources/translations/AdminLTEBundle.he.xliff:19]:
```suggestion
                <target>התחבר</target>
```

File: Resources/translations/AdminLTEBundle.he.xliff [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+                <source>Toggle navigation</source>
+                <target>Toggle navigation</target>
+            </trans-unit>
+            <trans-unit id="Remember Me">
+                <source>Remember Me</source>
+                <target>זכור אותי</target>
+            </trans-unit>
+            <trans-unit id="Sign in to start your session">
+                <source>Sign in to start your session</source>
+                <target>התחבר</target>
```
Comment (eransurg) [inline Resources/translations/AdminLTEBundle.he.xliff:15]: `<target>כניסה</target>`

Comment (eransurg) [inline Resources/translations/AdminLTEBundle.he.xliff:15]: `<target>כניסה</target>`

Comment (eransurg) [inline Resources/translations/AdminLTEBundle.he.xliff:15]:
```suggestion
                <target>כניסה</target>
```

**Comment by kevinpapst** [comment]
@eransurg good to go?

**Comment by eransurg** [comment]
Yes, just fixed two small mistakes.
Now it's good to go

**Comment by kevinpapst** [comment]
Awesome, thanks 👍

**Pull request merged**